src/components/RoarDataTable.vue

```diff
@@ -332,11 +332,32 @@
               </PvButton>
             </template>
           </PvColumn>
+          <PvColumn
```

Comment:
Is it possible to abstract this column out into a general-use button column? Actually, I believe I've already implemented a button column, you can check out Listusers.vue on main to see how it's implemented

Reply:
Done!